.gitignore

```diff
@@ -140,4 +140,4 @@ aiden-web/media/profile/*
 .DS_Store
 .env
 .python_version
-.secrets
+.secret
```

Comment:
What is the point of having a .env and a .secret file ?

Reply:
this is just for my own confort, because I hold my secrets in a `.secret` file (instead of .secretS which i previously added). Easier when you have to modify the `.env` file